CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- *!Breaking change*: Simplify configuration for the bundled apps
```

Comment:
Are all those changes related to this PR ?

Reply:
No I merged this PR https://github.com/giantswarm/observability-bundle/pull/130 and changelog was messed up